hello-world/README.md

```diff
@@ -0,0 +1,64 @@
+## Conclave Sample
+
+This is a sample to demonstrate the use of Persistent Filesystem and Persistent Map in your enclave. The Persistent
+Filesystem can be used to persist files and make them available even during enclave restarts. On the other hand, the
+persistent map is a persistent, encrypted key-value store that can be used to store information from your enclave. The
+enclave can use this key-value store to persist information as required. The enclave of this Conclave sample does three
+main tasks:
+
+1. Reverses the bytes that are passed in and send back the reversed string to the client.
+2. Stores the current input string to the Persistent Map under the key value: LAST-STRING. To demonstrate persistence,
+   also prints the input string from last request stored in the Persistent Map.
+3. Stores all input strings to a file in the Persistent FileSystem. Please refer to the `ReversePersistentEnclave.java`
+   for further details.
```

Comment:
I appreciate you've already written this sample but I think a more compelling example would be to store username credentials. User authentication is something people will want to do inside the enclave and will make the sample much more impactful.